File: src/main/java/com/nedap/retail/rheas/core/api/model/Spec.java
package com.nedap.retail.rheas.core.api.model;
public class Spec {

    public Integer id;
    public String name;
    public String[] event_types;

    public Spec(final int id, final String name, final String[] event_types) {
        this.id = id;
        this.name = name;
        this.event_types = event_types;
    }
    
    @Override
    public String toString() {
        StringBuilder result = new StringBuilder();
        result.append("id = ");
        result.append(this.id);
        result.append("\n");
        result.append("name = ");
        result.append(this.name);
        result.append("\n");
        result.append("event types = ");
        if (event_types!=null) {
            for (String e : event_types) {
                result.append(e);
                result.append(" ");
            }
        }
        return result.toString();
    }
}

File: csharp/Spec.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace StoreIDTopGateAPI
{
    [DataContract]
    class Spec
    {
        [DataMember]
        public int id { get; set; }
        [DataMember]
        public String name { get; set; }
        [DataMember]
        public String[] event_types { get; set; }

        public Spec(int id, String name, String[] event_types) {
            this.id = id;
            this.name = name;
            this.event_types = event_types;
        }
    
        public override String ToString() {
            StringBuilder result = new StringBuilder();
            result.Append("id = ");
            result.Append(this.id);
            result.Append("\n");
            result.Append("name = ");
            result.Append(this.name);
            result.Append("\n");
            result.Append("event types = ");
            if (event_types!=null) {
                foreach(String e in event_types) {
                    result.Append(e);
                    result.Append(" ");
                }
            }
            return result.ToString();
        }
    }
}
